src/no/runsafe/framework/api/server/IPlayerProvider.java

```diff
@@ -37,4 +37,10 @@
 	List<IPlayer> getOnlinePlayers();
 
 	List<String> getOnlinePlayers(IPlayer context, String playerName);
+
+	List<String> getUsernames(List<IPlayer> players);
+
+	List<IPlayer> getPlayersFromUniqueIds(List<UUID> playerIds);
```

Comment:
These two would look better as getPlayers and let type inference handle which one gets called